Cargo.toml

```diff
@@ -12,5 +12,5 @@ keywords = ["syn", "macros", "derive", "expand_substructure", "enum"]
 
 
 [dependencies]
-syn = "0.10"
+syn = "0.11"
```

Comment:
This kind of thing breaks Cargo and it's lets-use-the-latest-version-ever-all-the-time-even-if-it-introduces-multiple-versions-of-syn behaviour.